staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -254,6 +254,12 @@ type KubeletConfiguration struct {
 	// Default: true
 	// +optional
 	EnableDebuggingHandlers *bool `json:"enableDebuggingHandlers,omitempty"`
+	// enableProfiling enables profiling via web interface host:port/debug/pprof/
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
+	// enabling it may carry a performance impact.
+	// Default: false
+	// +optional
+	EnableProfiling bool `json:"enableProfiling,omitempty"`
```

Comment:
Isn't default value should be true? Otherwise there is some behavior change right? (/debug/pprof/* will stop working on existing clusters when they upgrade)

Reply:
I think "false" is ok.
 keep parity with control plane components.
There are risks to enable profile in the production environment.